docs/guides/create-decision-tables-using-dmn.md

```diff
@@ -0,0 +1,93 @@
+---
+id: create-decision-tables-using-dmn
+title: Create decision tables using DMN
+description: Model and execute decisions using a language both business analysts and developers can understand.
+keywords: [workflow, modeling]
+---
+
+<span class="badge badge--beginner">Beginner</span>
+<span class="badge badge--medium">Time estimate: 20 minutes</span>
+
+Decision Model and Notation (DMN) is a modeling approach owned by an institution called the Object Management Group ([OMG](https://www.omg.org/)), which also operates worldwide standards for [BPMN](./automating-a-process-using-bpmn.md).
+
+In [DMN](../components/modeler/dmn/dmn.md), decisions are modeled and executed using a language both business analysts and developers can understand. Model a set of rules within a table, and this will yield a decision to rapidly execute a process using a decision engine like Camunda.
+
+In this guide, we'll step through the lightweight implementation of a DMN diagram in [Camunda Platform 8](../components/concepts/what-is-camunda-platform-8.md), as both Camunda [Desktop](../components/modeler/desktop-modeler/install-the-modeler.md) and [Web Modeler](../components/modeler/about.md) both offer the same modeling experience for DMN 1.3 models.
+
+## Set up
+
+### Create a Camunda Platform 8 account
+
+We'll begin building our DMN diagrams with [Modeler](../components/modeler/about.md). To get started, ensure you've [created a Camunda Platform 8 account](./create-account.md).
+
+### Create a cluster
+
+import CreateCluster from './assets/react-components/create-cluster.md'
```

Comment:
Strange, this didn't render for me when I ran it locally. When this is merged, let's make sure we can see it.